src/stores/Frame/FrameStore.ts

```diff
@@ -2964,6 +2966,7 @@ export class FrameStore {
 
     @action setIsPreview = (isPreview: boolean) => {
         this.isPreview = isPreview;
+        this.distanceMeasuring = null;
```

Comment:
Do we need to check if `isPreview` is true? If `setIsPreview(false)` is called, do we enable distance tool?